apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -270,7 +275,8 @@ export class SendMessageEmail extends SendMessageBase {
       to: email,
       subject,
       html,
-      from: command.payload.$sender_email || integration?.credentials.from || 'no-reply@novu.co',
```

Comment:
I feel like this might be a breaking change, maybe we should not remove the `command.payload.$sender_email`?

Reply:
I searched for sender_email and could not find anywhere, not even in documents.. anyhow safe side I will add it back